Makefile

```diff
@@ -49,6 +43,9 @@ distserver: remove_old_config dist
 
 BUILT := ./built
 ADD-ON := add-on
+BUILT_ADD_ON := ./built/$(ADD-ON)
+DIST := ./dist
+XPI_NAME := loop@mozilla.org.xpi
```

Comment:
Could move  +XPI_FILE := built/$(XPI_NAME) up to here with the rest. Perhaps BUILT_XPI_FILE?